### DISCUSSION TITLE
Missing colours in 3.0.18

### DISCUSSION THREAD

**Discussion by clarg18** [tailwindlabs/tailwindcss#7325] [Help]
Hi,

I have `"tailwindcss": "^3.0.18",` and many colours are missing, such as `bg-green-200`, `bg-blue-300`.

My app.css looks like:

```
@tailwind base;
@tailwind components;
@tailwind utilities;
```

and tailwind config

```js
module.exports = {
    purge: [
        './resources/**/*.blade.php',
        './resources/**/*.js',
        './resources/**/*.vue',
    ],
  darkMode: false, // or 'media' or 'class'
  theme: {},
  variants: {
    extend: {},
  },
  plugins: [],
}
```

I added the green just to see if that was the issue.

```js
const colors = require('./node_modules/tailwindcss/colors');

module.exports = {
    purge: [
        './resources/**/*.blade.php',
        './resources/**/*.js',
        './resources/**/*.vue',
    ],
  darkMode: false, // or 'media' or 'class'
  theme: {
    extend: {
        colors: {
            green: colors.green,

        }
    },
  },
  variants: {
    extend: {},
  },
  plugins: [],

```

in both instances the only green I have is `text-green-600`

**Comment by adamwathan** [comment]
Can you share a reproduction? Sounds like a content configuration issue to me, where you aren’t properly scanning all of your templates.

**Reply by clarg18** [reply]
I'm sorry Adam but what do you mean by "share a reproduction"?

**Reply by adamwathan** [reply]
Hey no worries! I just mean create a repository on GitHub that includes a project that is set up to demonstrate the problem you're facing. Something I can clone down and run myself to see what's not working and look at what the problem might be.
